fluent/fluent.go

```diff
@@ -567,12 +567,12 @@ func (n *nextHopEntry) WithInterfaceRef(name string) *nextHopEntry {
 	return n
 }
 
-// WithInterfaceSubinterfaceRef specifies both an interface and a
-// specific subinterface to be used for the next-hop.
-func (n *nextHopEntry) WithInterfaceSubinterfaceRef(name string, sub uint64) *nextHopEntry {
+// WithSubInterfaceRef specifies both an interface and a specific
+// subinterface to be used for the next-hop.
+func (n *nextHopEntry) WithSubInterfaceRef(name string, subinterface uint64) *nextHopEntry {
```

Comment:
Just `WithSubinterface` would be my preference.

Reply:
Done